README.md

```diff
@@ -33,3 +33,27 @@ Include diagrams in the site:
 * Update the relevant markdown file to incorporate diagrams into the site.
 * 
 * `git commit` changes
+
+### Run the application locally
+
+Running this application is fairly manual at the moment and requires some technical expertise.  In summary, you can run the application on your own computer by downloading it to your computer, setting up a few dependencies and executing a number of shell commands to start desired components. You will need on your machine:-
```

Comment:
To "run" it locally you only need Docker and a JVM

Reply:
Cool